File: .kin-snapshot-tmp/tests/adapter-fixtures/csharp/Basic.cs
// Fixture: basic C# entities
// Expected: 1 class, 2 methods

using System;

public class Basic
{
    public string Name { get; }

    public Basic(string name)
    {
        Name = name;
    }

    public string GetName()
    {
        return Name;
    }

    private static int HelperInternal()
    {
        return 42;
    }
}

File: .kin-snapshot-tmp/tests/adapter-fixtures/php/basic.php
<?php
// SPDX-License-Identifier: Apache-2.0
// Copyright 2026 Firelock, LLC

// Fixture: basic PHP entities
// Expected: 1 class, 2 methods, 1 function

class UserService
{
    public function __construct(private string $name)
    {
    }

    public function displayName(): string
    {
        return $this->name;
    }
}

function helperInternal(string $name): string
{
    $svc = new UserService($name);
    return $svc->displayName();
}
